projects/storefrontlib/layout/a11y/keyboard-focus/persist/persist-focus.service.ts

```diff
@@ -43,7 +43,10 @@ export class PersistFocusService extends BaseFocusService {
    * Returns the group for the host element based on the configured group or
    * by the `data-cx-focus-group` attribute stored on the host.
    */
-  getPersistenceGroup(host: HTMLElement, config?: PersistFocusConfig): string {
-    return config?.group ? config.group : host.getAttribute(FOCUS_GROUP_ATTR);
+  getPersistenceGroup(
+    host: HTMLElement | undefined | null,
```

Comment:
I removed undefined from the type, and it still works. Why is it here?

Reply:
as reason for `host`

Reply:
makes sense! it ties to this here https://github.com/SAP/spartacus/pull/15658#discussion_r870762291 since we're saying to put | undefined instead